docs/src/getting-started/installation.md

```diff
@@ -18,7 +18,7 @@ rustup component add rustfmt
 See the solana [docs](https://docs.solana.com/cli/install-solana-cli-tools) for installation instructions. On macOS and Linux,
 
 ```bash
-sh -c "$(curl -sSfL https://release.solana.com/v1.7.8/install)"
+sh -c "$(curl -sSfL https://release.solana.com/v1.7.10/install)"
```

Comment:
The version in the docs should likely be consistent with the docker image published for each Anchor version https://github.com/project-serum/anchor/blob/master/docker/Makefile#L9.